TextDumpOutputer.h

```diff
@@ -7,19 +7,24 @@
 namespace cce::tf {
 class TextDumpOutputer : public OutputerBase {
  public:
- TextDumpOutputer() {}
+ TextDumpOutputer(bool perEventDump, bool summaryDump): perEventDump_(perEventDump), summaryDump_(summaryDump), eventCount_{0} {}
```

Comment:
`eventCount_` should be initialized before `perEventDump_`.

Reply:
curious that this wasn't caught by the compiler. Guess we need different warning settings?